CMakeLists.txt

```diff
@@ -80,6 +80,7 @@ if(USE_OPENSSL)
     -DUSE_OPENSSL_SRTP
     -DUSE_TLS
   )
+  include_directories( ${OPENSSL_INCLUDE_DIR} )
```

Comment:
looks like this one is needed for MacOS, other building fails because it cannot
find openssl headers ...